### PR TITLE
DOC: fixed using IP.prompt_manager which is removed from IPython 5.x

### LINKED ISSUE #14003: DOC: doc/make.py uses IPython promptManager, which is removed from IPython 5.x
When I build document as described in contributing guideline below,
http://pandas-docs.github.io/pandas-docs-travis/contributing.html#building-the-documentation

it makes an error like this.
```
AttributeError: "'InteractiveShell' object has no attribute 'prompt_manager'"
```

This file uses removed class, from IPython 5.x (promptManager)
http://ipython.readthedocs.io/en/stable/whatsnew/version5.html

file: doc/sphinxext/ipython_sphinxext/ipython_directive.py
```python
 805         if not self.state.document.current_source in self.seen_docs:
 806             self.shell.IP.history_manager.reset()
 807             self.shell.IP.execution_count = 1
 808             self.shell.IP.prompt_manager.width = 0
 809             self.seen_docs.add(self.state.document.current_source)
 810
```

#### output of ``pd.show_versions()``
INSTALLED VERSIONS
------------------
commit: a0d05dbf7477df863486d82a0cdd4e3e93023864
python: 3.5.2.final.0
python-bits: 64
OS: Darwin
OS-release: 15.3.0
machine: x86_64
processor: i386
byteorder: little
LC_ALL: None
LANG: ko_KR.UTF-8
LOCALE: ko_KR.UTF-8

pandas: 0.18.1+356.ga0d05db.dirty
nose: 1.3.7
pip: 8.1.2
setuptools: 25.1.6
Cython: 0.24.1
numpy: 1.11.1
scipy: 0.18.0
statsmodels: None
xarray: None
IPython: 5.1.0
sphinx: 1.4.1
patsy: None
dateutil: 2.5.3
pytz: 2016.6.1
blosc: None
bottleneck: None
tables: 3.2.3.1
numexpr: 2.6.1
matplotlib: 1.5.1
openpyxl: None
xlrd: None
xlwt: None
xlsxwriter: None
lxml: None
bs4: None
html5lib: None
httplib2: None
apiclient: None
sqlalchemy: None
pymysql: None
psycopg2: None
jinja2: 2.8
boto: None
pandas_datareader: None

I'm currently on pandas sprint at PyCon APAC, so i will send pull request today if possible.

### PR DESCRIPTION
**Pull request by znmean**
- [x] closes #14003, #13639
- [ ] tests added / passed (there's no test on building docs)
- [x] passes `git diff upstream/master | flake8 --diff`
- [ ] whatsnew entry (it's not user facing bug fix)

this PR was made during PyCon APAC, pandas sprint

thanks

### CODE REVIEW

File: doc/sphinxext/ipython_sphinxext/ipython_directive.py [hunk 1]
```diff
@@ -799,13 +799,19 @@ def setup(self):
...
 
-        # reset the execution count if we haven't processed this doc
-        #NOTE: this may be borked if there are multiple seen_doc tmp files
-        #check time stamp?
-        if not self.state.document.current_source in self.seen_docs:
+        """
+        reset the execution count if we haven't processed this doc
+        NOTE: this may be borked if there are multiple seen_doc tmp files
+        check time stamp?
+        """
```
Comment (jorisvandenbossche) [inline doc/sphinxext/ipython_sphinxext/ipython_directive.py]: can you leave this as `#` comments? (that's how multiline comments in the codebase are done `"""` is used for docstrings)

**Comment by jorisvandenbossche** [comment]
Thanks! 
Added a small comment. I also think a whatsnew notice is not needed, as it is not a user facing change/bug fix

**Comment by znmean** [comment]
Thanks! Done what you said.
- revert to # comment
- no whatsnew notice

**Comment by jorisvandenbossche** [comment]
@znmean Thanks!

**Pull request merged**